src/native/libs/System.Globalization.Native/pal_locale.h

```diff
@@ -9,6 +9,8 @@ PALEXPORT int32_t GlobalizationNative_GetLocales(UChar *value, int32_t valueLeng
 
 PALEXPORT int32_t GlobalizationNative_GetLocaleName(const UChar* localeName, UChar* value, int32_t valueLength);
 
+PALEXPORT const char* NativeGetLocaleName(const char* localeName, int32_t valueLength);
```

Comment:
This should still have `GlobalizationNative_` prefix to follow the coding conventions. 

https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/interop-guidelines.md#unix-shims:  all exports should have a prefix that corresponds to the Libraries' name, e.g. "SystemNative_" or "CryptoNative_"